engine-comparison/startup-runner.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+mkdir -p ~/input
+BENCHMARK=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/benchmark -H "Metadata-Flavor: Google")
+FENGINE_CONFIG=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/fengine -H "Metadata-Flavor: Google")
```

Comment:
Yeah, this is a little confusing... the crux of the issue is that, at this point, we don't have metadata.cfg; this is the minimum amount of information needed to download metadata.cfg, as well as all other files.

It's also not precisely the same data, as metadata.cfg has FENGINE_CONFIG rather than FENGINE_NAME. In general, this code feels suboptimal, I'll work on this